### PR TITLE
Juanciro/order code

### PR DESCRIPTION
**Pull request by Ciroye**
- Moved the python codes that were in the root  (count_records.py and show_tfrecord.py) to the "galvasr2" folder
- Created a folder inside "galvasr2" with the codelabs
- Organized the environments; leaving only yours and adding pyspark
- Don't make any code changes

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -1,13 +1,13 @@
-name: 100k-hours-lingvo
```
Comment (galv) [inline environment.yml:1]: Change this part of docker/galvasr2.dockerfile from this:

```
COPY environment2.yml /install/environment2.yml
ENV PATH="/install/miniconda3/bin/:${PATH}"
RUN conda env create -f /install/environment2.yml
```

to this:

```
COPY environment.yml /install/environment.yml
ENV PATH="/install/miniconda3/bin/:${PATH}"
RUN conda env create -f /install/environment.yml
```

File: galvasr2/align/spark/schemas.py [hunk 2]
```diff
@@ -45,7 +45,7 @@
             StructField("aspect_ratio", StringType(), True),
             StructField("backup_location", StringType(), True),
             StructField("closed_captioning", StringType(), True),
-            StructField("collection", ArrayType(StringType(), True), True),
+            StructField("collection", StringType(), True),
```
Comment (galv) [inline galvasr2/align/spark/schemas.py:48]: I thought you had already made this change in this past PR: https://github.com/mlcommons/peoples-speech/pull/8/files#diff-49bd79597919909effba2db47b8ad03e3f7a3645f28ae9113571e0a5e700dd79R48

Are you sure that `juanciro/order-code` is rebased on top of the main branch? Contact me privately if you're not sure what rebasing means.

**Review by galv — CHANGES_REQUESTED** [review]
It looks like your branch juanciro/order-code contains the commits from juanciro/dump_cc_by_licenses.

Try doing this:

```
git fetch origin
git checkout -b juanciro/order-code-2 origin/main
git cherry-pick a522b991d231f8f23f9038a2917b64f999540f95
```

Then push your branch and make a new pull request (I also believe you can edit an existing pull request to change the branch, but I'm not sure).

**Comment by Ciroye** [comment]
Open a new PR with correct configuration

**Pull request closed without merging**